code/game/objects/items/grenades/plastic.dm

```diff
@@ -93,6 +93,9 @@
 	if(bomb_target != user && HAS_TRAIT(user, TRAIT_PACIFISM) && isliving(bomb_target))
 		to_chat(user, span_warning("You don't want to harm other living beings!"))
 		return
+	if(isobserver(bomb_target))
```

Comment:
check for invisibility instead? people could put c4 on hidden revenants, or supermatter timers etc